include/grpc/impl/codegen/port_platform.h

```diff
@@ -290,6 +290,12 @@
 #endif
 #endif /* GPR_NO_AUTODETECT_PLATFORM */
 
+#if defined(__has_include)
```

Comment:
Is `#if defined` valid C89? I thought we've been typically avoiding it's use in `include/grpc`